packages/botfuel-dialog/src/messages/message.js

```diff
@@ -27,7 +27,7 @@ class Message extends ValidObject {
    * @param {*} value - the message value
    * @param {Object} [options] - the message options
    */
-  constructor(type, sender, value, options) {
+  constructor(type, sender, value, options = null) {
```

Comment:
Because when we are building the JSON message we are using this for options:
```javascript
...(!!this.options && { options: this.options })
```
and 
```javascript
!!{} === true
```
but
```javascript
!!null === false
```